test/examples/test_simple_gp_regression.py

```diff
@@ -374,6 +374,14 @@ def test_posterior_latent_gp_and_likelihood_fast_pred_var(self, cuda=False):
 
             self.assertLess(torch.max(var_diff / noise), 0.05)
 
+            if cuda:
```

Comment:
Tests like these are guaranteed to be flakey. Let's remove this for now, and address it when we tackle performance tests (#406)